Gopkg.toml

```diff
@@ -0,0 +1,46 @@
+
+# Gopkg.toml example
+#
+# Refer to https://github.com/golang/dep/blob/master/docs/Gopkg.toml.md
+# for detailed Gopkg.toml documentation.
+#
+# required = ["github.com/user/thing/cmd/thing"]
+# ignored = ["github.com/user/project/pkgX", "bitbucket.org/user/project/pkgA/pkgY"]
+#
+# [[constraint]]
+#   name = "github.com/user/project"
+#   version = "1.0.0"
+#
+# [[constraint]]
+#   name = "github.com/user/project2"
+#   branch = "dev"
+#   source = "github.com/myfork/project2"
+#
+# [[override]]
+#  name = "github.com/x/y"
+#  version = "2.4.0"
+
+
+[[constraint]]
+  branch = "master"
+  name = "github.com/golang/glog"
+
+[[constraint]]
+  branch = "master"
+  name = "github.com/golang/protobuf"
+
+[[constraint]]
+  branch = "master"
+  name = "github.com/kylelemons/godebug"
+
+[[constraint]]
+  branch = "master"
+  name = "github.com/openconfig/gnmi"
+
+[[constraint]]
+  branch = "master"
+  name = "github.com/openconfig/goyang"
```

Comment:
We may want consider controlling the versions for goyang and gnmi at least. That way if their behavior changes it will be easy to go back in time to the right compatible versions (should we ever need to do that).

Reply:
Yeah, I think this would be worth considering. Right now, I don't expect either changes to be backwards incompatible, but we should determine whether it's still worth fixing the versions. 

The tests use `go get` rather than `dep ensure` so would be decoupled from this. I wonder whether it's possible for us to get travis to run a set of tests that must pass (using `dep ensure`) and then some informational smoke tests with unfixed versions and pulling the OpenConfig schema.